docs/web3.txpool.rst

```diff
@@ -21,7 +21,7 @@ The following properties are available on the ``web3.txpool`` namespace.
     * Delegates to ``txpool_inspect`` RPC Method
 
     Returns a textual summary of all transactions currently pending for
-    inclusing in the next block(s) as will as ones that are scheduled for
+    inclusing in the next block(s) as well as ones that are scheduled for
```

Comment:
While we're at it, I think `inclusing` should be `including` here

Reply:
Fixed 👍